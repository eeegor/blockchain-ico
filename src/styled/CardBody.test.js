import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import { CardBody } from '.';

describe('CardBody', () => {
	test('renders card body', () => {
		const tree = renderer.create(<CardBody>Hello</CardBody>).toJSON();
		expect(tree).toMatchSnapshot();
	});

	test('renders card body with custom padding', () => {
		const tree = renderer
			.create(<CardBody paddingY={30}>Hello</CardBody>)
			.toJSON();
		expect(tree).toMatchSnapshot();
		expect(tree).toHaveStyleRule('padding-top', '30px !important');
		expect(tree).toHaveStyleRule('padding-bottom', '30px !important');
	});
});
