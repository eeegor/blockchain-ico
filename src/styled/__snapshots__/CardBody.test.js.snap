// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardBody renders card body 1`] = `
<div
  className=""
>
  Hello
</div>
`;

exports[`CardBody renders card body with custom padding 1`] = `
.c0 {
  padding-top: 30px !important;
  padding-bottom: 30px !important;
}

<div
  className="c0"
>
  Hello
</div>
`;
